packages/dashboard-core-plugins/src/panels/NotebookPanel.tsx

```diff
@@ -600,6 +607,41 @@ class NotebookPanel extends Component<NotebookPanelProps, NotebookPanelState> {
     this.setState({ showCloseModal: false });
   }
 
+  /**
+   * Close overwritten tabs
+   * @params {string} fileName = The name of the file to be overwritten
```

Comment:
You shouldn't need the `{string}` part here, since it's already specified in the TypeScript. May need to check your VS Code settings if that was automatically added for you when adding the JS Doc snippet.
```suggestion
   * @params fileName The name of the file to be overwritten
```